platform_versioned_docs/version-24.1/monitoring/dashboard.mdx

```diff
@@ -25,5 +27,27 @@ Depending on the filter selected, each card details a separate workspace or orga
 
 For a detailed view of runs, you can do one of the following:
 
-- Select a run integer value in the table to navigate to a run list filtered by the status and time range selected.
-- Select a workspace name in the table to navigate to a run list filtered by the workspace selected.
+- Select a pipeline run integer value in the table to navigate to a pipeline run list filtered by the status and time range selected.
+- Select a workspace name in the table to navigate to a pipeline run list filtered by the workspace selected.
+
+## Data Studios
+
+You can explore the status of data studio sessions in your organizational workspaces. On the **Dashboard** page, select **Data Studios**. The following statuses are listed with the number of data studio sessions in each status:
+
+- `Running`
+- `Errored`
+- `Starting`
+- `Stopping`
+- `Stopped`
+
+### Filters and summary
+
+The **Dashboard** view defaults to all organizations and workspaces you can access. Select the **View** dropdown menu to filter by organizations and workspaces. Select a status in the table to navigate to a list filtered by the status selected.
+
+## Export data
```

Comment:
I think this needs to be indented to 3 times again because there's another `Export data` entry above (which I missed first time around)
```suggestion
### Export data
```